### PR TITLE
Let CD depends on CI

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-149

### CODE REVIEW

File: .github/workflows/trigger-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Run CI
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+on: [push, pull_request]
```
Comment (pkong-ds) [inline .github/workflows/trigger-pr.yaml]: trigger-pr should only need `pull_request`?

I suppose `ci.yaml` will run twice `on push` with this configuration, which is undesired?
1. in `trigger-cicd.yaml`
2. in `trigger-pr.yaml`

```suggestion
on: [pull_request]
```

File: .github/workflows/trigger-cicd.yaml
Comment (pkong-ds) [inline .github/workflows/trigger-cicd.yaml:1]: discussion: filename should follow same pattern? pick one col below?

| name by what it does         | name by what triggers it                                   |
|---------------------|------------------------------------------|
| trigger-cicd.yaml (or trigger-cd.yaml ?)                 | trigger-push.yaml                            |
| trigger-ci.yaml                | trigger-pr.yaml                            |

**Review by pkong-ds — CHANGES_REQUESTED** [review]
Did you reference from https://github.com/oursky/mtr-oneclick/tree/main/.github/workflows ? 😆

**Comment by YayunHuang** [comment]
@pkong-ds updated, and I pick `trigger-pr`, `trigger-push`~~

**Review by pkong-ds — APPROVED** [review]

**Pull request merged**